packages/dicomImageLoader/src/imageLoader/isColorConversionRequirementsFulfilled.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * This function checks color space conversion data requirements before
+ * apply them
+ * @param imageFrame
+ * @param RGBA
+ * @returns
+ */
+export default function isColorConversionRequirementsFulfilled(
+  imageFrame,
+  RGBA
+) {
+  const { rows, columns } = imageFrame;
+  if (imageFrame.photometricInterpretation === 'RGB') {
+    return imageFrame.length % 3 === 0;
+  } else if (imageFrame.photometricInterpretation === 'YBR_RCT') {
```

Comment:
Is there a specific reference available that can provide an explanation for these concepts? If the photometric is YBR_RCT and the frame is a multiple of 3, can we confidently say that it was converted to RGB in the decoder?

Note: I'm not very familiar with color spaces, so this might be a commonly known fact.

Reply:
I just got all checks in the correspondent functions and put them here. In future we can create better checks here.

Reply:
I will limit the check to only YBR_FULL_422